src/app/compose/compose.component.ts

```diff
@@ -59,6 +59,8 @@ export class ComposeComponent implements AfterViewInit, OnDestroy, OnInit {
     public uploadprogress: number = null;
     public uploadingFiles: File[] = null;
     public saved: Date = null;
+    has_pasted_signature;
+    signature;
```

Comment:
Can we give these some types? I assume the latter might be `any` anyway, but maybe we can do better than plain JS here :)